waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -21,8 +21,12 @@ type SpamHandler* = proc(wakuMessage: WakuMessage): void {.gcsafe, closure,
 
 # membership contract interface
 contract(MembershipContract):
-  # TODO define a return type of bool for register method to signify a successful registration
   proc register(pubkey: Uint256) # external payable
+  proc MemberRegistered(pubkey: Uint256, index: Uint256) {.event.}
```

Comment:
```suggestion
  proc memberRegistered(pubkey: Uint256, index: Uint256) {.event.}
```

Reply:
Thanks for the suggestion, though, it is not possible to change it as it needs to exactly match the name of the corresponding even on the contract. The reason is that when communicating with the contract the hash of the corresponding event i.e., `MemberRegistered` will be used to filter the transaction logs.